apps/include/apps.h

```diff
@@ -292,16 +212,15 @@ ASN1_VALUE *app_http_post_asn1(const char *host, const char *port,
  */
 # define SERIAL_RAND_BITS        159
 
-int app_isdir(const char *);
-int app_access(const char *, int flag);
-int fileno_stdin(void);
-int fileno_stdout(void);
-int raw_read_stdin(void *, int);
-int raw_write_stdout(const void *, int);
+//int app_access(const char *, int flag);
```

Comment:
**Concerning the naming:** `apps_extracted` was just intended as a temporary name. I assumed, there are already suitable files where the functions/definitions should be put into and after having moved them to the "final" files, I wanted to delete `apps_extracted` completely. After having done the separation, I have the impression the functions should not be in the same file. I didn't try to find out what every function is used for, so it's hard to decide for me where they really should belong to. I'd be happy to have input about where I should put which function best.
`os_wrappers` is probably a better name, I think we should still prefix is with `apps_` because there are only functions inside being relevant for apps and the functions are (partially) also named like `app_`. Without the prefix, I would think the file would be the place for "general" os-wrappers (which I think is generally a good idea, but introducing that should be done in a different PR).
**Concerning the comments:** I always planned to make a cleanup, but as a last step. Until now it helped me to track where which parts I took from which file when I made additional movements.